src/plot_api/plot_api.js

```diff
@@ -3714,6 +3714,9 @@ function purge(gd) {
 function makePlotFramework(gd) {
     var gd3 = d3.select(gd);
     var fullLayout = gd._fullLayout;
+    if(fullLayout._inverseTransform === undefined) {
+        fullLayout._inverseTransform = Lib.inverseTransformMatrix(Lib.getFullTransformMatrix(gd));
```

Comment:
Let's also compute and stash `._inverseScale(X|Y)` here and use that from other places.
```js
fullLayout._inverseScaleX = Math.sqrt(m[0][0] * m[0][0] + m[0][1] * m[0][1] + m[0][2] * m[0][2]);
fullLayout._inverseScaleY = Math.sqrt(m[1][0] * m[1][0] + m[1][1] * m[1][1] + m[1][2] * m[1][2]);
```
Thanks!